### PR TITLE
Improve disposal, /bigtree and /fireball (#2461) @montlikadani

### PR DESCRIPTION
**Pull request by montlikadani**
https://github.com/EssentialsX/Essentials/commit/a2a95ed840a130f5589526a7b8d93707390b3ffa
Other:
- Fix last commit not specified darkoak to plugin.yml
- Added splashpotion & lingeringpotion to throws items with `/fireball` command

_If you accept this PR, I will add the string to other messages file._

### CODE REVIEW

**Comment by Ichbinjoe** [comment]
Not to be pedantic, but this PR adds one 'feature' and fixes 2 missing items for 1.13.

It would be nice if these were split out into 3 separate PRs and approved in that way. At the very least, splitting the inventory title out from the rest would be nice.  As you mentioned, the PR still needs the message in the rest of the i18n files

**Comment by mdcfe** [comment]
@Ichbinjoe The messages only need adding to `messages.properties` - Crowdin is used for the rest.

**Comment by montlikadani** [comment]
@Ichbinjoe 
I don't understand what needs 3 PR separately, but if really need it, I will do it.

What @md678685 said the messages can be modified on Crowdin, even if only add messages to the messages.properties file.

👍 Make PR | 👎 No, we stay here

**Comment by Ichbinjoe** [comment]
I guess I would only really want the PRs split up if it was going to block one thing or another. I guess everything looks good. I think I misunderstood how crowdin handles i18n, so you don't need to handle the translations here.

**Review by Ichbinjoe — DISMISSED** [review]

File: Essentials/src/messages.properties [hunk 1]
```diff
@@ -102,6 +102,7 @@ destinationNotSet=Destination not set\!
 disabled=disabled
 disabledToSpawnMob=\u00a74Spawning this mob was disabled in the config file.
 disableUnlimited=\u00a76Disabled unlimited placing of\u00a7c {0} \u00a76for {1}.
+disposal=\u00a7cDisposal
```
Comment (mdcfe) [inline Essentials/src/messages.properties]: Should this have a colour by default? It didn't have a colour before.

**Review by mdcfe — CHANGES_REQUESTED** [review]
The inventory created by `[Disposal]` sign should also use the translated message. Also, I don't think the default message should have a colour. Everything else in the PR is fine.

File: Essentials/src/messages.properties (hunk 1 above)
Comment (montlikadani) [reply Essentials/src/messages.properties]: Ok, but we need GUI formatting like colors. On the sign, there is no need for the translated string, so we use it without it.

Comment (md678685) [reply Essentials/src/messages.properties]: I was referring to the inventory title created here: https://github.com/EssentialsX/Essentials/blob/849efa9756495a34260d57ff1d81df1283e5f5e7/Essentials/src/com/earth2me/essentials/signs/SignDisposal.java#L14

Comment (mdcfe) [reply Essentials/src/messages.properties]: I was referring to the inventory title created here: https://github.com/EssentialsX/Essentials/blob/849efa9756495a34260d57ff1d81df1283e5f5e7/Essentials/src/com/earth2me/essentials/signs/SignDisposal.java#L14

The title of the inventory was the default before - we should keep the normal grey colour (no colour code) as the default.

**Review by mdcfe — DISMISSED** [review]

**Comment by montlikadani** [comment]
@Ichbinjoe What are you waiting for author? I'm already fixed the last changes required.

**Comment by Ichbinjoe** [comment]
@montlikadani I'm holding this PR because the Disposal string is colored, which would be a behavior change (as mentioned in the comments above).

If @md678685 is fine with the colored change, I'm fine with it to, but otherwise we should keep it consistent with past behavior and let the user change it on their own.

**Comment by montlikadani** [comment]
I think we can allow color disposal GUI to let users use colors. If a user don't want colored, then remove it itself.

**Review by mdcfe — DISMISSED** [review]

**Review by mdcfe — APPROVED** [review]

**Comment by Andre601** [comment]
Wouldn't it be an option to allow players manually rename the disposal's inv title throught the sign directly?

Like the first line is `[disposal]` and second line (or perhaps all 3 lines) would contain the name of the inventory title.

**Comment by mdcfe** [comment]
@Andre601 Feel free to open a feature request for it. I'm wary of this though as it could be used to trick unsuspecting players (say, if someone calls their disposal inventory "Vault" or "Chest" etc).

**Comment by Andre601** [comment]
> @Andre601 Feel free to open a feature request for it. I'm wary of this though as it could be used to trick unsuspecting players (say, if someone calls their disposal inventory "Vault" or "Chest" etc).

I don't think that would be the case because the `[disposal]` part would still exist and if players can read (which they should) then they would realize that this is a disposal-inventory.

And by opening a feature request you mean making an issue or try making a PR (Second would be more difficult to do for me but easier for you to check and implement)

**Comment by mdcfe** [comment]
> by opening a feature request you mean making an issue or try making a PR 

I meant open an issue, but you could try making a PR if you want 😛

**Pull request merged**